EvidenceApi.Tests/V1/UseCase/SendNotificationUploadConfirmationToResidentAndStaffTests.cs

```diff
@@ -13,13 +13,13 @@
 
 namespace EvidenceApi.Tests.V1.UseCase
 {
-    public class SendNotificationUploadConfirmationForResidentTests
```

Comment:
The use case and tests were renamed to take into account that it's sending notifications to both Resident and Staff